pro/src/components/Callout/Callout.module.scss

```diff
@@ -8,6 +8,7 @@
   color: var(--color-black);
   border-radius: rem.torem(8px);
   padding: rem.torem(16px);
+  line-height: rem.torem(20px);
```

Comment:
Le changement va impacter tous les callout de l'app, c'est ce qui est voulu ?
Et d'ailleurs, ce n'est pas censé être mis dans le system design ? @GuillaumeMgz  au cas où

Reply:
Franchement il faut vraiment avoir un oeil de lynx pour voir la différence. And yet, j'ai voulu être maniaque et faire la modif en me basant sur le composant Figma.
Il s'agit simplement de faire en sorte que les textes soient bien alignés avec les icônes de taille 20.

Reply:
La line-height doit rester celle des typos à part si c'est pas possible! On essaie de tout aligner sur les mixins qui sont dans `form.scss` pour que quand on plug les variables du DS tout change en même temps. 

Si il faut aligner l'icon, possible de donner une hauteur de la tailel de la line-height au container de l'icon ?